### PR TITLE
express-get-json

### PR DESCRIPTION
**Pull request by ChorongShin**
Screen Shot 2022-07-01 at 5 17 06 PM

### CODE REVIEW

File: express-get-json/index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  }
+};
+
+const app = express();
+
+app.get('/api/grades', (req, res) => {
+
+  res.json([grades]);
+
+});
```
Comment (TimelikeClosure) [inline express-get-json/index.js:30]: Your server's `GET /api/grades` response should be an array of objects, not an array of an object of objects:

---

target reference

---

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by ChorongShin** [comment]
Screen Shot 2022-07-01 at 5 29 39 PM

File: express-get-json/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+const arr = [];
+
+app.get('/api/grades', (req, res) => {
+  for (const property in grades) {
+    arr.push(grades[property]);
+  }
+
+  res.json(arr);
+
+});
```
Comment (TimelikeClosure) [inline express-get-json/index.js:35]: Please make sure that every `GET` request results in the same number of grades in the response.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by ChorongShin** [comment]
Screen Shot 2022-07-01 at 5 44 53 PM

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**